oc/pod.py

```diff
@@ -82,16 +82,17 @@ def exec_command(self, **kw):
         check_ec = kw.get('check_ec', True)
         long_running = kw.get('long_running', False)
 
-        cmd_obj = CmdObj(cmd, timeout, wait, check_ec, long_running)
+        cmd_obj = CmdObj(
```

Comment:
I believe according to our guidelines in the case where a line like this is too long (over 80 chars) you would split the parenthesis like you have here but each argument should be on it's own line.

For example:
```
cmd_obj = CmdObj(
            cmd,
            timeout,
            wait,
            check_ec,
            long_running,
)
```
Maybe @RazTamir or @petr-balogh can confirm this for me.

Reply:
yeah correct